webplugin/template/chat.html

```diff
@@ -38,9 +48,21 @@
                                 iframe.classList.add("change-kommunicate-iframe-height");
                             });
                         });
+                        //to show the pre chat lead collection Form if enabled 
+                        var preChatLauncher = window.KommunicateGlobal.window.document.getElementById("km-anonymous-chat-launcher")
+                        if (window.KommunicateGlobal && preChatLauncher) {
+                            var intervalForPreChatLead = setTimeout(function () {
+                                preChatLauncher.click()
+                                window.KommunicateGlobal.window.document.getElementById("km-modal-close").style.display = "none"
+                            }, 500);
+                        }
+                        window.KommunicateGlobal && preChatLauncher &&
+                            preChatLauncher.addEventListener("click", function () {
+                                { clearInterval(intervalForPreChatLead) }
+                            })
                         closeButton.parentElement.style.display = "none";
                         Kommunicate.launchConversation();
-                        KommunicateGlobal.document.getElementById('mck-sidebox-launcher').click();
+                        // KommunicateGlobal.document.getElementById('mck-sidebox-launcher').click();
```

Comment:
will this work for normal use case
ie were prechat lead form is not there?

Reply:
yes, I checked!